src/index.js

```diff
@@ -1,9 +1,5 @@
-import { createFactory } from '@volusion/element-block-utils';
-import { StarterBlockFactory } from './Block';
-import { getStyles } from './getStyles';
-import { getConfigSchema, defaultConfig } from './configs';
+import { StarterBlock as block } from './Block';
+import { configSchema, defaultConfig } from './configs';
```

Comment:
**Question:** `configSchema` is the new name we're utilizing from now on?

Reply:
Yes. With the new contract, we can import `ElementPropTypes` directly, so `configSchema` is now an object, not a function.